common/internal_model/src/main/scala/weco/catalogue/internal_model/work/Relations.scala

```diff
@@ -66,20 +66,20 @@ object RelationSet {
     existingRelations: List[Relation],
     newRelations: List[Relation]
   ): List[Relation] = {
-    val newTitles =
-      newRelations.map(relation => removeTerminalPunctuation(relation.title))
-
-    // duplicated if it shares a title, but not if it already has an identifier.
-    def isRelationDuplicated(relation: Relation): Boolean =
-      relation.id.isEmpty && newTitles.contains(
-        removeTerminalPunctuation(relation.title)
-      )
-
-    val relationsToKeep =
-      existingRelations.filter(r => !isRelationDuplicated(r))
```

Comment:
Previously: keep if it is not duplicated:  i.e.  not(identified AND same name exists)